test/cpp/end2end/async_end2end_test.cc

```diff
@@ -486,7 +486,7 @@ TEST_P(AsyncEnd2endTest, DoThenAsyncNextRpc) {
   auto recv_resp_ptr = &recv_response;
   auto status_ptr = &recv_status;
   send_response.set_message(recv_request.message());
-  auto lambda_3 = [&, this, resp_writer_ptr, send_response]() {
+  auto lambda_3 = [&, resp_writer_ptr, send_response]() {
     resp_writer_ptr->Finish(send_response, Status::OK, tag(3));
```

Comment:
Actually, this one is bad for another reason. We really shouldn't have anonymous captures.

Reply:
Done.